docs/source/telegram.games-tree.rst

```diff
@@ -1,6 +1,19 @@
 Games
 -----
 
+Your bot can offer users **HTML5 games** to play solo or to compete against each other in groups and one-on-one chats. Create games via `@BotFather <https://telegram.me/BotFather>`_ using the /*newgame* command. Please note that this kind of power requires responsibility: you will need to accept the terms for each game that your bots will be offering.
+
+* Games are a new type of content on Telegram, represented by the :doc:`Game <telegram.game>` and :doc:`InlineQueryResultGame <telegram.inlinequeryresultgame>` objects.
```

Comment:
<code>:doc:\`Game <telegram.game>\`</code>

This is better, but is has two downsides:

* by using `:doc:`, Sphinx doesn't know that you're referencing a class. Indexing and rendering might be wrong. Please use `:class:` for classes `:meth:` for methods of classes etc. The available types are listed at https://www.sphinx-doc.org/en/master/usage/domains/python.html#python-xref-roles (drop the leading `:py`) and are also used throughout the PTB documentation
* you make your live harder by repeating "game" twice. If you write just <code>:class:\`telegram.Game\`</code> you aovid duplication and have correcting rendering.